ui/src/alerting/actions/checks.ts

```diff
@@ -101,14 +106,16 @@ export const getChecks = () => async (
 }
 
 export const getCheckForTimeMachine = (checkID: string) => async (
-  dispatch: Dispatch<TimeMachineAction | NotificationAction>,
+  dispatch: Dispatch<
```

Comment:
Since most actions don't take arguments- I don't really see why we should restrict which actions a thunk can dispatch? I'm really even fine with not typing dispatch at all. Maybe I'm missing something but I just don't see what there is to gain? It's not like it's a strong preference, but that is where I'm at. Open to argument.